src/coreclr/src/gc/gc.h

```diff
@@ -89,7 +89,7 @@ enum gc_etw_segment_type
 {
     gc_etw_segment_small_object_heap = 0,
     gc_etw_segment_large_object_heap = 1,
-    gc_etw_segment_read_only_heap = 2
+    gc_etw_segment_read_only_heap = 2,
```

Comment:
>y_heap = 2, [](start = 27, length = 11)

why this change...?

Reply:
I am guessing it is from adding one more item and then removing. This is not intentional.